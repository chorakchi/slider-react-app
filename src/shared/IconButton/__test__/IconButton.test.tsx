
import React from "react";
import { shallow, mount } from "enzyme";
import * as renderer from "react-test-renderer";
import {IconButton} from "./../index";

describe("IconButton", () => {
  const defaultProps = {
    onClick: () => {},
  };

  it("should render the IconButton", () => {
    const subject = shallow(<IconButton {...defaultProps} />);
    expect(subject).toHaveLength(1);
  });

  it("should render correctly and match to the Snapshot", () => {
    const subject = renderer.create(<IconButton {...defaultProps} />).toJSON();
    expect(subject).toMatchSnapshot();
  });
});