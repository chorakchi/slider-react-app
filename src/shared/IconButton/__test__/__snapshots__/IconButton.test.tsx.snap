// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconButton should render correctly and match to the Snapshot 1`] = `
<div
  className="sc-bdnylx bMWfWs"
  onClick={[Function]}
/>
`;
